### DISCUSSION TITLE
Engine freezes at random, leaving me with 10+ unfair misses when playing a mod

### DISCUSSION THREAD

**Discussion by NightrayGH** [KadeArchive/Kade-Engine#1240] [Q&A]
The engine just freezes at random, usually ending up killing my run. This has happened on Matt (WiiK 2), Tricky v2.0 and 2.1, the Camellia mod, and others. I would really appreciate help on finding a fix.

**Comment by Gostar64** [comment]
It usually happens from the game loading or overloading
I know a fix in hellclown, where you just hit 7 and put 4 fire notes at the same time on Tricky's side

Run from the bob mod would make the arrows overload, idk what happens in Wiik 2 and Camellia though
